src/components/sponsorships/sponsorships.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import styled from 'styled-components'
+import Layout from '../../components/layout'
+import Tabs from 'react-responsive-tabs'
+import LegislatorList from '../../pages/all-legislators/LegislatorList'
+
+const TabsContaner = styled.div`
+  background-color: #eaecef;
+  padding: 1rem;
+`
+const Container = styled.div`
+  padding: 1rem 8rem;
+`
+
+export default function SponsoredBill({
+  pageContext: { billData: bill, houseSponsors, senateSponsors, sponsors },
+}) {
+  const tabItems = [
+    {
+      title: 'House Reps',
+      component: <LegislatorList data={houseSponsors} chamber='lower' sessionNumber={'193rd'} />,
```

Comment:
This will never show the scores in the column on the right because the necessary fields (`recordedVotePercentage` and `score`) are not included in the records in `houseSponsors`. I also noticed that the legislators are sorted alphabetically by first name. There may be a way we can reuse more of the code that puts this data together for the main `all-legislators` page.

Reply:
fixed!